src/js/static-pages/sass/static-pages.scss

```diff
@@ -6,6 +6,11 @@
 @import "layouts/l-playbook";
 
 // ----- Modules that only affect non-React pages. ---- //
+@import "modules/m-card";
```

Comment:
These are all playbook styles or for features that are only used in the static pages. I searched for the classes from each file and did not find other uses.